planning/scenario_selector/src/scenario_selector_node/scenario_selector_node.cpp

```diff
@@ -392,17 +392,6 @@ ScenarioSelectorNode::ScenarioSelectorNode(const rclcpp::NodeOptions & node_opti
 
   timer_ = rclcpp::create_timer(
     this, get_clock(), period_ns, std::bind(&ScenarioSelectorNode::onTimer, this));
-
-  // Wait for first tf
-  while (rclcpp::ok()) {
-    try {
-      tf_buffer_.lookupTransform("map", "base_link", tf2::TimePointZero);
-      break;
-    } catch (tf2::TransformException & ex) {
-      RCLCPP_DEBUG(this->get_logger(), "waiting for initial pose...");
-      rclcpp::sleep_for(std::chrono::milliseconds(100));
-    }
-  }
```

Comment:
I think this should be changed in the `feat` pull request.

Reply:
Thank you!
I made a [PR](https://github.com/autowarefoundation/autoware.universe/pull/3414) for this part.

Reply:
@kyoichi-sugahara Thank you. Could you update the description so that others can understand the background of the pull request? :pray:

Reply:
@kenji-miyake 
Yes I updated description!